packages/@react-spectrum/s2/stories/DatePicker.stories.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {Button, Content, ContextualHelp, DatePicker, Footer, Form, Heading, Link, Text} from '../src';
+import {categorizeArgTypes} from './utils';
+import {fn} from '@storybook/test';
+import type {Meta, StoryObj} from '@storybook/react';
+import {style} from '../style' with {type: 'macro'};
+
+const meta: Meta<typeof DatePicker> = {
+  component: DatePicker,
+  parameters: {
+    layout: 'centered'
+  },
+  tags: ['autodocs'],
+  argTypes: {
+    ...categorizeArgTypes('Events', ['onChange']),
+    label: {control: {type: 'text'}},
+    description: {control: {type: 'text'}},
+    errorMessage: {control: {type: 'text'}},
+    contextualHelp: {table: {disable: true}}
+  },
+  args: {
+    onOpenChange: fn(),
+    onChange: fn()
+  },
+  title: 'DatePicker'
+};
+
+export default meta;
+type Story = StoryObj<typeof DatePicker>;
+
+export const Example: Story = {
+  args: {
+    'aria-label': 'Birthday'
```

Comment:
Should we show a visual label by default? Seems like that's usually our recommendation